kep/260-network-aware-scheduling/README.md

```diff
@@ -0,0 +1,1432 @@
+        (...) // If the node being scored and the pod's hostname is the same node -> sum = sum + 0
+        (...) // If Nodes belong to the same zone -> sum = sum + 1
+        (...) // Otherwise, retrieve the cost from the map and add to sum -> sum = sum + cost
+
+    // 7) Return: Accumulated cost as score
+    score = sum
+    return score, framework.NewStatus(framework.Success, "Accumulated cost added as score, normalization ensures lower costs are favored")
+}
+```
+
+Then, we get the maximum and minimum costs for all candidate nodes to normalize the values between 0 and 100. 
+After normalization, **nodes with lower costs are favored** since it also corresponds to lower latency:
+
+```go
+// NormalizeScore : normalize scores
+func (pl *NetworkOverhead) NormalizeScore(ctx context.Context, state *framework.CycleState, pod *v1.Pod, scores framework.NodeScoreList) *framework.Status {
+	// Lower scores correspond to lower latency
+	// 1) Get Min and Max Scores to normalize between framework.MaxNodeScore and framework.MinNodeScore
+	(...)
+	// 2) If all nodes were given the minimum score, return
+    (...)
+    // 3) Loop: for all scores 
+        (...) // If max != min
+            (...) // node_normalized_cost = MAX_SCORE * ( ( nodeScore - minCost) / (maxCost - minCost)
+            (...) // nodeScore = MAX_SCORE - node_normalized_cost
+        (...) // Else: If maxCost = minCost, avoid division by 0
+            (...) // node_normalized_cost = MAX_SCORE * - minCost
+            (...) // nodeScore = MAX_SCORE - node_normalized_cost
+
+    // 4) Return after the loop finishes 
+    return nil
+}
+```
+
+<!-- previous normalization:
+    // Lower scores correspond to lower latency
+   	// Get Min and Max Scores to normalize between framework.MaxNodeScore and framework.MinNodeScore
+   	minCost, maxCost := GetMinMaxScores(scores)
+   
+   	// If all nodes were given the minimum score, return
+   	if minCost == 0 && maxCost == 0 {
+   		return nil
+   	}
+   
+   	var normCost float64
+   	for i := range scores {
+   		if maxCost != minCost { // If max != min
+   			// node_normalized_cost = MAX_SCORE * ( ( nodeScore - minCost) / (maxCost - minCost)
+   			// nodeScore = MAX_SCORE - node_normalized_cost
+   			normCost = float64(framework.MaxNodeScore) * float64(scores[i].Score-minCost) / float64(maxCost-minCost)
+   			scores[i].Score = framework.MaxNodeScore - int64(normCost)
+   		} else { // If maxCost = minCost, avoid division by 0
+   			normCost = float64(scores[i].Score - minCost)
+   			scores[i].Score = framework.MaxNodeScore - int64(normCost)
+   		}
+   	}
+   	return nil
+-->
+
+We plan to combine our scoring plugin with other scoring plugins (e.g., `BalancedAllocation`, `LeastRequestedPriority`, etc). 
+We will attribute a higher weight to our plugin to prefer decisions focused on low latency. 
+For instance, consider the following scheduler policy config file as an example:
+
+```json
```

Comment:
Scheduler policy has been removed in latest ComponentConfig, can you port this to a ComponentConfig example? Or, just simply delete it.

Reply:
Thanks for spotting the old config definition. We have replaced it with an example of a scheduler config yaml file. We can remove the yaml and add just the link if it is preferred.